<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertWindow</class>
 <widget class="QWidget" name="InsertWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="basicBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>761</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>BasicInfo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>726</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="namelabel">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameIn"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="agelabel">
       <property name="text">
        <string>Age:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="ageIn">
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="genderlabel">
       <property name="text">
        <string>Gender:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="birthdaylabel">
       <property name="text">
        <string>Birthday:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDateEdit" name="birthdayIn">
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="emaillabel">
       <property name="text">
        <string>Email:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="emailIn">
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="phonelabel">
       <property name="text">
        <string>Phone:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QLineEdit" name="phoneIn"/>
     </item>
     <item row="0" column="5">
      <widget class="QComboBox" name="genderBox">
       <item>
        <property name="text">
         <string>Male</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Female</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="extraBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>601</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>AdditionalInfo</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>581</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="schoollabel">
       <property name="text">
        <string>School:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="schoolIn"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="courselabel">
       <property name="text">
        <string>Course:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="courseIn"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="companylabel">
       <property name="text">
        <string>Company:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="companyIn"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="postlabel">
       <property name="text">
        <string>Post:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="postIn"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="calllabel">
       <property name="text">
        <string>Call:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="callIn"/>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="placelabel">
       <property name="text">
        <string>Place:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="placeIn">
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>place where you met...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="remarkBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>601</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Remarks</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QTextEdit" name="remarkIn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>581</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="relationlabel">
      <property name="text">
       <string>Relation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="relationCombo">
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
      <item>
       <property name="text">
        <string>classmates</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>friends</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>colleagues</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>relatives</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>teachers</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>superiors</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>clients</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>others</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
